scripts/StartMenu.gd

```diff
@@ -2,7 +2,7 @@ extends Control
 
 func _on_StartGame_pressed():
     print("start game pressed")
-    get_tree().change_scene("scenes/Level1.tscn")
+    get_tree().change_scene("scenes/world/World.tscn")
```

Comment:
Hmmm, maybe we should use absolute paths here? Like `res://scenes/world/World.tscn`

Reply:
You're right. Will fix that.